pull-requests.md

```diff
@@ -31,26 +41,19 @@ If anyone has engaged in behavior that is potentially illegal or makes you or so
 
 ## Opening a pull request
 
-### If you find something you want to work on
-
-Ideally you'll have found an issue or project task to work on before opening a pull request. Every pull request opened should be to close an existing issue. If you haven't check through issues on repos or public project boards to find something. If you have found a problem on MDN open an issue - more [information on issues can be found here](). **Issues need to be triaged before work can start.**
-
-Make sure the issue isn't assigned and no one has already opened a pull request for the task. You can do that by checking the issue and searching open pull requests
-
-If you're not sure where to start, reach out to us on [Discord]() and ask for feedback before starting on any work.
-
-### Opening a pull request
-
 When you're ready to open a pull request, follow these guidelines:
 
-- **PRs should be short and focused to one issue:** If possible, group related set of changes into multiple, small PRs. If a PR becomes too large, the reviewer may close it and ask that you to submit pull requests for each logical set of changes that belong together.
-- **Add a description of what the pull request is doing:** Provide as much context and reason for opening the pull request.
-- **Add the link to the issue you are closing or working on:** Add 'Fixes' or 'Relates to' and the related issue in the description of the pull request. See [github link here]()
-- **Accompany code example changes with content changes:** This is important to ensure that updated examples are served correctly. If content changes also affect how examples are used, the related code examples should be updated in associated repositories.
-- **Add 'depends on'** with a link to a dependency if there are PRs that must land first (e.g. linked examples in other repos).
-- **Add a reviewer** if you know who should review your PR already, such as a team member or a topic owner.
+- **Add the link to the issue you are closing:** Add 'Fixes' or 'Relates to' and the related issue in the description of the pull request.
+  More information about issues can be found [in our Community guidelines pages](https://developer.mozilla.org/en-US/docs/MDN/Community/Issues).
```

Comment:
Should we instead link here to GitHub's guidelines for using the keywords in PR description?

```suggestion
  More information about linking to issues in pull requests can be found in [GitHub docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
```